cvmfs/cvmfs.cc

```diff
@@ -1523,6 +1523,33 @@ static void cvmfs_getxattr(fuse_req_t req, fuse_ino_t ino, const char *name,
       fuse_reply_err(req, ENOATTR);
       return;
     }
+  } else if (attr == "user.chunks_access") {
```

Comment:
The work `access` sounds like permission control, how about `chunks_info`?

Reply:
How about chunks_list?

Reply:
Yep, I think that's fine.  Perhaps `chunk_list`.